### ISSUE TITLE
How can I use my own JSON data from remote server?

### ISSUE THREAD

**Issue by n0490b** [kvkirthy/VenCKI-Samples#2]
I have a database that is being queried (php + mysql) that is returning JSON data is it possible to connect it with this?

**Comment by kvkirthy** [comment]
Yes. Which same were you referring to? There are 4/5 samples in this repo.
